### ISSUE TITLE
Hystrix dashboard

### ISSUE THREAD

**Issue by gsogol** [hudl/Mjolnir#58]
Does this actually plug into the hystrix dashboard?

**Comment by robhruska** [comment]
It doesn't, the library's more of an "inspired-by Hystrix" and not so much a direct port of it.

**Comment by gsogol** [comment]
Got you. I know there's a way to integrate into the dashboard. That would be killer. 

The other important aspect is the collapser. But got you.

**Comment by robhruska** [comment]
Yeah, Hystrix's collapser's pretty sweet. I might argue that it's out of scope for Mjolnir - I'm actually kind of interested in why Netflix didn't build that into Ribbon instead of Hystrix, since it feels more like a transport optimization and not a fast-failure concern.

**Comment by gsogol** [comment]
True
